src/gc/gc.d

```diff
@@ -17,6 +17,10 @@ module gc.gc;
 
 /************** Debugging ***************************/
 
+// Note: Some of these need to be set globally
+// (for the entire Druntime) in the Makefile
+// instead of uncommenting the lines here.
+
```

Comment:
AFAICT it's only SENTINEL that needs to be set globally to disable some unittests.